### DISCUSSION TITLE
[closed][x][manuscript][blueprint][demo] Should we handle chromosome labels somehow (e.g. chr1 == I...)?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#103] [Team Posts]
Not even sure what is the status of this issue in the current human reference assemblies

**Comment by dzerbino** [comment]
GRC and Ensembl have always stood by 1 and UCSC by chr1. Because of UCSC influence (esp. in the US research community) GRC have started producing files with chr1. In other words, the human research world is split in two. 

Traditionally, UCSC users refer to genome assemblies by their UCSC names (hg38 instead of GRCh38) so this disambiguated what coordinate system they were using.

**Comment by jmfernandez** [comment]
Each assembly at NCBI has file like next:

ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/vertebrate_mammalian/Homo_sapiens/all_assembly_versions/GCA_000001405.15_GRCh38/GCA_000001405.15_GRCh38_assembly_report+ucsc_names.txt

which contains all the correspondences between both worlds of chromosomic naming. The first column are the chromosomes, scaffolds, etc... without the prefix, and the last column is the UCSC-like name for that chromosome, scaffold, etc....

So, to be more FAIR, we could need to map the common assembly names (hg38, GRCh38) to the official ones (GCA_000001405.15 and GCA_000001405.15_GRCh38). You can have a look at the "History" section of https://www.ncbi.nlm.nih.gov/assembly/?term=GCA_000001405

**Comment by sveinugu** [comment]
Cool! I did not know about those files, that is very useful. But as we are working on metadata, I don't think we should handle any such conversions in this project. But a follow-up project could be to provide chromosome name conversion, together with liftOver functionality, which would actually be very useful for serving old track collections.

But as far as I understand, the official assembly name is GRCh38, or one of the patches (which are typically not used for tracks, as a patch does not change the coordinate system). the GCA_ ones are accession number for the sequences, right?

The question remains, should we make sure that we use "GRCh38" if the files follow the official naming 1, 2, ..., and "hg38" if the files follow chr1, chr2, ... We do actually have a "genome build predictor" tool (https://hyperbrowser.uio.no/refgenome/), which can be used for testing this, but it is only for manual use at this point (no REST API). Also, the current version only supports UCSC-style naming, but it can easily be extended to support both types. Obviously a possibility for the future. It is from the paper "Genome build information is an essential part of genomic track files" https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1312-1.

The HyperBrowser only supports UCSC-style names, and had an issue in the demo with the BLUEPRINT data, as those were tagged with "GRCh38". Changing the genome to "hg38" fixed the issue, with no change in the data files, which means that the data files are actually following UCSC-style names.

Btw: there are no roman numbers in the human genomes, as far as I remember, so the title of this discussion point in misleading. But there are roman numbers in other genomes..

**Comment by sveinugu** [comment]
@dzerbino As the last post was very long, let me just be clear with my suggestion: I think we should change the genome assembly in the BLUEPRINT data to "hg38". Discuss!

**Comment by dzerbino** [comment]
Sure, that's a solution.

**Comment by sveinugu** [comment]
Handled by https://github.com/fairtracks/fairtracks_standard/pull/33. Closing this

**Comment by sveinugu** [comment]
Added link to this in topic https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/41 for the manuscript.
